tests/improver-nbhood/01-help.bats

```diff
@@ -43,7 +43,8 @@ plugin to a file with one cube.
 
 positional arguments:
   NEIGHBOURHOOD_METHOD  The neighbourhood method to apply in neighbourhood
-                        processing. Options: "circular", "square".
+                        processing. Options: "circular_probabilities",
```

Comment:
This is now pretty long as a required argument. Could we have something like:
`--shape (circular or square, default circular)`
`--percentiles (process as percentiles rather than probabilities (default False)`